src/github-api-helper.ts

```diff
@@ -125,19 +125,14 @@ async function downloadArchive(
   commit: string,
   baseUrl?: string
 ): Promise<Buffer> {
-  const octokit = getOctokit(authToken, {baseUrl: baseUrl})
-  const params: Octokit.ReposGetArchiveLinkParams = {
+  const octokit = github.getOctokit(authToken, {baseUrl: baseUrl})
+  const download = IS_WINDOWS
+    ? octokit.rest.repos.downloadZipballArchive
+    : octokit.rest.repos.downloadTarballArchive
+  const response = await download({
```

Comment:
No need for handling the exception with something like `response.status != 200`. If the request fails, an exception is thrown and handled upstream.

![image](https://user-images.githubusercontent.com/31069338/231449236-0c5c968f-cae6-42cb-ba4f-48be268748d5.png)